Web api minimal hosting model doc migration (#10231)
auth0/docs: articles/quickstart/backend/aspnet-core-webapi/01-authorization.md

2. Make a call to the `AddJwtBearer` method to register the JWT Bearer authentication scheme. Configure your Auth0 domain as the authority, and your Auth0 API identifier as the audience. In some cases the access token will not have a `sub` claim which will lead to `User.Identity.Name` being `null`. If you want to map a different claim to `User.Identity.Name` then add it to `options.TokenValidationParameters` within the `AddAuthentication()` call.

```csharp
// Program.cs

> var domain = $"https://{builder.Configuration["Auth0:Domain"]}/";

Reviewer: Can we add what builder is here ? So let's add `var builder = WebApplication.CreateBuilder(args);`

Author: Done as per [501677e](https://github.com/auth0/docs/pull/10231/commits/501677e349cf36e85c17f6bd1c4a5c96d19ea441).

builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
.AddJwtBearer(options =>
{
    options.Authority = domain;
    options.Audience = builder.Configuration["Auth0:Audience"];
    options.TokenValidationParameters = new TokenValidationParameters
    {
        NameClaimType = ClaimTypes.NameIdentifier
    };
});
```

> To add the authentication and authorization middleware to the middleware pipeline, call  the `UseAuthentication` and `UseAuthorization` methods under your Program.cs `var app = builder.Build();` method:

Reviewer: ```suggestion
To add the authentication and authorization middleware to the middleware pipeline, add a call to the `UseAuthentication` and `UseAuthorization` methods in your Program.cs file:
```

```csharp

> // Startup.cs

Reviewer: ```suggestion
// Program.cs
```

> app.UseAuthentication();

Reviewer: Can we add what `app` is here ?

Author: You mean like add that `var app = builder.build();` statement? 
I did so as per [0a5e178](https://github.com/auth0/docs/pull/10231/commits/0a5e178ac640d1b904f06ef94a61648900e677d3). Let me know if I missed anything.

}
```

> Under your Program's `var builder = WebApplication.CreateBuilder(args);` method, call the `AddAuthorization` method. To add policies for the scopes, call `AddPolicy` for each scope. Also ensure that you register the `HasScopeHandler` as a singleton:

Reviewer: ```suggestion
In your Program.cs file, add a call to the `AddAuthorization` method. To add policies for the scopes, call `AddPolicy` for each scope. Also ensure that you register the `HasScopeHandler` as a singleton:
```
